digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/middleware/TrafficSignManager.scala

```diff
@@ -32,23 +39,51 @@ object TrafficSignManager {
 case class TrafficSignManager(manoeuvreService: ManoeuvreService, roadLinkService: RoadLinkService) {
   def withDynTransaction[T](f: => T): T = OracleDatabase.withDynTransaction(f)
 
+  case object TrafficSignSerializer extends CustomSerializer[TrafficSignProperty](format =>
```

Comment:
can these serializers be put on the JsonSerializer.scala and then imported? this one seems to be identical with the other one